README.md

```diff
@@ -65,3 +65,11 @@
 * New Features - Days Until Sell Out Prediction
 * New Features - User Event Watchlist
 * New Features - Customized User Event Digest
+
+Instructions:

```

Comment:
I would try to call out this section more, so it is easier for users to find.  In addition, it should be written so more generally.  How does a user run this app if they don't have an API key?  How do they create their own?

In addition, you should include instructions for setting up the python environment.  I would recommend virtualenv over conda.  The instructions can be similar to:
- run virtualenv -p python3 my_proj
- run source my_proj/bin/activate
- run pip install -r requirements.txt

Reply:
That makes sense, I will incorporate that into the next sprint.